Makefile

```diff
@@ -107,6 +107,34 @@ install:
 
 # Browser-specific hosts targets
 
+.PHONY: hosts-iridium
+hosts-iridium:
+	@case $(OS) in \
+	Linux)      mkdir -p "/etc/iridium-browser/native-messaging-hosts/"; \
+	            ln -sfv "$(LIB_DIR)/browserpass/hosts/chromium/$(APP_ID).json" "/etc/iridium-browser/native-messaging-hosts/$(APP_ID).json"; \
+	            [ -e "/etc/iridium-browser/native-messaging-hosts/$(APP_ID).json" ] || echo "Error: the symlink points to a non-existent location" >&2; \
+	            ;; \
+	Darwin)     mkdir -p "/Library/Application Support/Chromium/NativeMessagingHosts/"; \
```

Comment:
Just so I know in the future, is this path for macOS a guess or a tested fact? 🙂 Judging by Vivaldi and Brave, there's a complete and utter mess with these paths in Chromium forks...